app/models/event.rb

```diff
@@ -258,6 +258,19 @@ def course_kind?
     kind_class == Event::Kind && kind.present?
   end
 
+  def ical_events
```

Comment:
Instead of putting everything into model files (fat model anti-pattern), create a specific class in `app/domain/export/ical`. This is where export formats for different models are already handled.